client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -34,57 +33,64 @@ const months = [
   'November',
   'December',
 ];
-const days = [...Array(30).keys()].map((i) => i + 1);
+const days = [...Array(31).keys()].map((i) => i + 1);
 const years = [...Array(119).keys()].map((i) => i + 1903).sort((a, b) => b - a);
 
 export default function EditProfileTab(): JSX.Element {
   const classes = useStyles();
-  const [accountType, setAccountType] = useState<UserProfile['accountType'] | null>('partner');
-  const [available, setAvailable] = useState<UserProfile['available'] | null>(false);
-  const [availability, setAvailability] = useState<UserProfile['availability'] | null>('availability');
-  const [firstName, setFirstName] = useState<UserProfile['firstName'] | null>();
-  const [lastName, setLastName] = useState<UserProfile['lastName'] | null>();
-  const [gender, setGender] = useState<UserProfile['gender'] | null>('gender');
-  const [birthdateMonth, setBirthdateMonth] = useState<UserProfile['birthdateMonth'] | null>('month');
-  const [birthdateDay, setBirthdateDay] = useState<UserProfile['birthdateDay'] | null>('day');
-  const [birthdateYear, setBirthdateYear] = useState<UserProfile['birthdateYear'] | null>('year');
-  const [email, setEmail] = useState<UserProfile['email'] | null>();
-  const [phone, setPhone] = useState<UserProfile['phone'] | null>();
-  const [address, setAddress] = useState<UserProfile['address'] | null>();
-  const [description, setDescription] = useState<UserProfile['description'] | null>();
+  const [accountType, setAccountType] = useState<string>('partner');
   const [showPhoneInput, setShowPhoneInput] = useState(false);
+  const [profile, setProfile] = useState<ProfileInterface>({
+    firstName: '',
+    lastName: '',
+    description: '',
+    address: '',
+    phoneNumber: '',
+    dateOfBirth: new Date('December 17, 1995 03:24:00'),
+    available: false,
+    availability: '',
+    gender: '',
+    email: '',
+  });
+  const userId = '619c1eb37a1e963a5b179c4b';
+
+  useEffect(() => {
+    fetchProfile(userId).then((data) => setProfile(data.success.profile)); //get profileValues and set to profile state
+  }, []);
 
   const formik = useFormik({
     initialValues: {
-      available: false,
-      availability: 'availability',
-      firstName: '',
-      lastName: '',
-      gender: 'gender',
-      birthdateMonth: 'month',
-      birthdateDay: 'day',
-      birthdateYear: 'year',
-      email: '',
-      phone: '',
-      address: '',
-      description: '',
+      available: profile.available,
+      availability: profile.availability || 'availability',
```

Comment:
```suggestion
      availability: profile.availability || '',
```
Won't this be an empty string here? Ditto for all the initial values

Reply:
No, I had to implement a workaround to display placeholders inside the Material UI Select components. The form is prefilled with the value the user previously chose for their profile, or it shows the placeholder.